packages/server/src/index.ts

```
@@ -2,47 +2,54 @@ import { createServer } from 'node:http';
+    plugins: [adminContextPlugin(), useGraphQLModules(application)],
     context: (yogaContext): AccounterContext => {
+      const headers = yogaContext.request.headers;
+      const authRole = headers.get('x-auth-role');
+      const authUsername = headers.get('x-auth-username');
+      const authUserId = headers.get('x-auth-user-id');
+      if (!authRole || !authUsername || !authUserId) {
+        throw new Error('Unauthorized');
+      }
+      const currentUser: UserType = {
+        userId: authUserId,
+        role: authRole as any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)